packages/e2e-tests/specs/site-editor/template-revert.test.js

```diff
@@ -164,15 +168,14 @@ describe( 'Template Revert', () => {
 		expect( contentBefore ).toBe( contentAfter );
 	} );
 
-	it.skip( 'should show the original content after revert, clicking undo then redo in the header toolbar', async () => {
+	it( 'should show the original content after revert, clicking undo then redo in the header toolbar', async () => {
```

Comment:
Other changes in this file are not relevant to the PR, but refactored a bit (correct function signatures, etc..).